labs/firefly.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: FireFly
+parent: Labs
+---
+# Lab Name
+FireFly
+
+# Short Description
+FireFly is a multiparty system for enterprise data flows, powered by blockchain.
+
+# Scope of Lab
+FireFly is designed to make building blockchain apps quick and easy, by allowing developers to focus on business logic, instead of infrastructure and cryptography. Enterprise use cases often require a mix of off-chain and on-chain activities to implement a use case end-to-end. FireFly provides a set of pre-integrated runtimes and an API to build event-driven, multi-party applications with a combination of off-chain data flows and on-chain transactions.
+
+FireFly is designed to be completely modular, allowing components at each layer to be swapped out for a different implementation, including the messaging system, the database, and even the blockchain itself.
+
+
+# Initial Committers
+- https://github.com/peterbroadhurst
+- https://github.com/nguyer
+- https://github.com/gabriel-indik
+- https://github.com/shorsher
+- https://github.com/awrichar
+
```

Comment:
In the template, there is a section for Sponsor. Can you add this section and note that you do not have a sponsor?

Reply:
I do not see the string "Sponsor" in this file anywhere: https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/main/proposal-template.md

Am I missing something?

Reply:
My bad. Looks like Ry removed it 4 days ago.

Reply:
No worries! Thanks!

Reply:
> My bad. Looks like Ry removed it 4 days ago.

I think that's a mistake actually. We said the Sponsor was no longer required but we didn't eliminate the role.
Note: this has nothing to do with Firefly but I think the Sponsor section should be put back into the template. @ryjones ?

Reply:
@lehors OK, sorry, I'm out over my skis. I'll put it back